### PR TITLE
2703: Fix landing page visible for halloaschaffenburg

### PR DESCRIPTION
**Pull request by sarahsporck**
### Short description

Do not render landing route for halloaschaffenburg.

### Proposed changes

- Render landing route conditionally
- Remove unecessary redirect

### Resolved issues

Fixes: #2703 

---

### CODE REVIEW

File: web/src/RootSwitcher.tsx [hunk 1]
```diff
@@ -84,11 +84,8 @@ const RootSwitcher = ({ setContentLanguage }: RootSwitcherProps): ReactElement =
 
         {/* Redirects */}
         <Route path='/' element={<Navigate to={fixedCityPath ?? landingPath} replace />} />
-        <Route path={LANDING_ROUTE} element={<Navigate to={fixedCityPath ?? landingPath} replace />} />
+        <Route path={RoutePatterns[LANDING_ROUTE]} element={<Navigate to={fixedCityPath ?? landingPath} replace />} />
```
Comment (steffenkleinle) [inline web/src/RootSwitcher.tsx:87]: Hmmm... While this works and there is no endless loop, we are basically redirecting from `/landing/de` to `/landing/de` if there is no fixed city. I'd prefer if we could avoid that by only having that redirect if there is a fixed city.
```suggestion
        {!!fixedCityPath && (
          <Route path={RoutePatterns[LANDING_ROUTE]} element={<Navigate to={fixedCityPath} replace />} />
        )}
```

File: web/src/RootSwitcher.tsx [hunk 2]
```diff
@@ -84,11 +84,8 @@ const RootSwitcher = ({ setContentLanguage }: RootSwitcherProps): ReactElement =
...
         <Route path='/:cityCode' element={<Navigate to={fixedCityPath ?? language} replace />} />
```
Comment (steffenkleinle) [inline web/src/RootSwitcher.tsx:88]: This now also handles redirecting `/landing`. Perhaps we should make that clear using a comment :see_no_evil:

**Review by steffenkleinle — APPROVED** [review]
Tested on firefox :wrench:

**Review by LeandraH — APPROVED** [review]
😍  Nice! Tested in Firefox

**Pull request merged**